docs/source/compiler-delegate-and-partitioner.md

```diff
@@ -1,4 +1,4 @@
-# Backend and Delegate
+# Backends and Delegates
```

Comment:
Can we move this to "Backend Development" subsection from "Compiler Entry Points"?